### PR TITLE
Removed console.log and useless functions

### PR DESCRIPTION
**Pull request by BrahimS**

### CODE REVIEW

File: frontend/src/composables/WorkSpace/useDomainData.js [hunk 1]
```diff
@@ -101,12 +98,8 @@ export default function useDomainData() {
 	};
 
 	const addTagtoDomain = async (tag) => {
-		try {
 			await API.post(`/domain/${choosenNodeID.value}/tag/${tag.id}`);
 			refreshDomainTag(choosenNodeID.value);
-		} catch (error) {
-			console.log(err);
```
Comment (Mari0nV) [inline frontend/src/composables/WorkSpace/useDomainData.js:108]: Could be good to add error handling later on! What is happening for now if the API call is in error? Should we write a US to handle this error?

File: frontend/src/composables/WorkSpace/useDomainData.js [hunk 2]
```diff
@@ -148,10 +141,8 @@ export default function useDomainData() {
 	getTagsTree();
 
 	const OnDelete = async (id) => {
-		console.log("deleted ID:", id);
 		await API.delete(`/domain/${choosenNodeID.value}/tag/${id}`)
 			.then(() => refreshDomainTag(choosenNodeID.value))
-			.catch((error) => console.log(error));
```
Comment (Mari0nV) [inline frontend/src/composables/WorkSpace/useDomainData.js:154]: Could be good to add error handling later on! What is happening for now if the API call is in error? Should we write a US to handle this error?

File: frontend/src/composables/WorkSpace/useDomainData.js [hunk 3]
```diff
@@ -168,9 +159,6 @@ export default function useDomainData() {
 			.onOk(() => {
 				OnDelete(id);
 			})
-			.onCancel(() => {
-				console.log("Cancel");
```
Comment (Mari0nV) [inline frontend/src/composables/WorkSpace/useDomainData.js:172]: I've seen a lot of onCancel methods that do nothing. What is happening in the interface if a user cancel an operation? Should we write a US to handle this kind of behavior?

File: frontend/src/pages/Home/PageHomeProducts.vue [hunk 4]
```diff
@@ -94,8 +94,7 @@ export default defineComponent({
 		const oepnDialog = ref(false);
 
 		const executeGit = async () => {
-			let exc = await store.dispatch("appGit/execute");
-			console.log("exc: ", exc);
+			 await store.dispatch("appGit/execute");
```
Comment (Mari0nV) [inline frontend/src/pages/Home/PageHomeProducts.vue]: It seems that there is an extra space before await

File: frontend/src/pages/Home/PageHomeProfile.vue [hunk 5]
```diff
@@ -326,18 +324,13 @@ export default {
 			formData.append("id", imagesUID);
 			formData.append("file", file[0], file[0].name);
 			await API.post(`/file/${imagesUID}`, formData).then((res) => {
```
Comment (Mari0nV) [inline frontend/src/pages/Home/PageHomeProfile.vue]: shouldn't the ".then((rest) => {}" be deleted as well?

**Review by Mari0nV — CHANGES_REQUESTED** [review]
Good code cleaning!
There are an extra space to remove and a *then* to remove.
I also have a few questions regarding things to handle later.

File: frontend/src/composables/WorkSpace/useDomainData.js (hunk 1 above)
Comment (BrahimS) [reply frontend/src/composables/WorkSpace/useDomainData.js:108]: Yes all the API calls will be handled globally on the axios middleware

File: frontend/src/composables/WorkSpace/useDomainData.js (hunk 2 above)
Comment (BrahimS) [reply frontend/src/composables/WorkSpace/useDomainData.js:154]: Yes all the API calls will be handled globally on the axios middleware

File: frontend/src/composables/WorkSpace/useDomainData.js (hunk 3 above)
Comment (BrahimS) [reply frontend/src/composables/WorkSpace/useDomainData.js:172]: it's done

File: frontend/src/pages/Home/PageHomeProducts.vue (hunk 4 above)
Comment (BrahimS) [reply frontend/src/pages/Home/PageHomeProducts.vue]: Oups 😄

File: frontend/src/pages/Home/PageHomeProfile.vue (hunk 5 above)
Comment (BrahimS) [reply frontend/src/pages/Home/PageHomeProfile.vue]: it's done

File: frontend/src/composables/WorkSpace/useDomainData.js (hunk 3 above)
Comment (Mari0nV) [reply frontend/src/composables/WorkSpace/useDomainData.js:172]: I've seen you deleted them yes, but I asked if we need to handle cancellation and therefore write a User Story about it?

**Review by Mari0nV — APPROVED** [review]
Looks good for me!

**Pull request merged**